presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -64,6 +64,13 @@ class TaskManager {
           std::unordered_map<std::string, std::string>>&&
           connectorConfigStrings);
 
+  // Iterates through a map of resultRequests and fetch data from
```

Comment:
typo: fetch -> fetches

Would you explain how this method handles the case when task has already finished and there is no buffer in the POBM.